vendor/engines/secure_rooms/app/services/secure_rooms/access_handlers/occupancy_handler.rb

```diff
@@ -27,7 +27,7 @@ def process
           )
         else
           # TODO: Add error cases
-          raise NotImplementedError
+          Rails.logger.warn "Encountered Double-Scan Situation"
```

Comment:
yeah I figured I'm mainly unblocking and no one will actually read this message between now and actual implementation